app.rb

```diff
@@ -2,8 +2,9 @@
 require_relative 'teacher'
 require_relative 'student'
 require_relative 'rental'
+require_relative 'user_inputs'
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/201698543-72312559-900d-448d-a081-661065dc1227.png)

- [ ] Your methods should be SOLID means single responsible but you can see that your methods are checking the input(First Responsibility), taking the inputs(Second Responsibility), creating the like in this case creating a person class(Third Responsibility) from taken inputs. :thinking: Kindly make all your methods SOLID :+1: